src/Symfony/Component/Console/Output/Output.php

```diff
@@ -140,7 +140,8 @@ public function writeln($messages, $options = self::OUTPUT_NORMAL)
     public function write($messages, $newline = false, $options = self::OUTPUT_NORMAL)
     {
         $messages = (array) $messages;
-
+        $options = is_numeric($options) ? (int) $options : 0;
```

Comment:
// vendor/doctrine/doctrine-bundle/Command/ImportMappingDoctrineCommand.php
...
protected function execute(InputInterface $input, OutputInterface $output)
    {
	...
        if ($metadata) {
    	    ...	
        } else {
            $output->writeln('Database does not have any mapping information.', 'ERROR');
            $output->writeln('', 'ERROR');
	    ...
        }
...
For example, in this particular case, an exception is thrown instead of the message text.